js/libs/keycloak-admin-client/src/utils/auth.ts

```diff
@@ -77,10 +77,8 @@ export const getToken = async (settings: Settings): Promise<TokenResponse> => {
   const headers = new Headers(options.headers);
 
   if (credentials.clientSecret) {
-    headers.set(
-      "Authorization",
-      atob(credentials.clientId + ":" + credentials.clientSecret)
-    );
+    const authHeaderValue = `Basic ${btoa(`${credentials.clientId}:${credentials.clientSecret}`)}`;
+    headers.set("Authorization", authHeaderValue);
```

Comment:
I see no need to put this in a variable if it's only going to get used right after.
```suggestion
    headers.set("Authorization", `Basic ${btoa(`${credentials.clientId}:${credentials.clientSecret}`)}`);
```